### DISCUSSION TITLE
How can i disable this behavior? Weird Highlighting upon Component creation

### DISCUSSION THREAD

**Discussion by escapingSamsara** [LazyVim/LazyVim#4334] [Q&A]
Hello, i currently re-installed neovim to update to v0.10 and reinstalled the new lazyvimstarter. upon writing in react, i cam across a (for me) very annoying issue, where the creation of a component is highlighting the whole component, until i change buffer, it is super annoying and wasnt happening on my older neovim/lazyvim version. how can i disable this? procedure: opening new .tsx file, creating the component via: rafce snippet. upon entering the name of the component and then pressing esc/enter, the component gets highlighted, and the highlighting does not go away until i switch buffer for example . (no i am not highlighting it via visual mode or anything)

![highlighted component](https://github.com/user-attachments/assets/e0e313a6-60a7-45bd-8429-ff9b27ed8506)

**Comment by dpetka2001** [comment]
This comes from Neovim's new `vim.snippet` engine and it should be reported there. To my understanding this happens because it thinks you are still inside the snippet session and it can be removed by pressing `<Tab>` until you exit the snippet session. Unfortunately, I'm not that familiar with Neovim's codebase myself and this is just a guess from my part.
